src/components/UserMenu.js

```diff
@@ -85,26 +85,34 @@ export function UserMenu({ onUserInputsChange }) {
                         </Col>
                     </Form.Group>
 
-                    <Form.Group>
-                        <Form.Check
-                            type="switch"
-                            id="residentInScotland"
-                            label="Resident in Scotland"
-                            name="residentInScotland"
-                            checked={inputs.residentInScotland}
-                            onChange={handleInputChange}
-                        />
-                        <Form.Check
-                            type="switch"
-                            id="noNI"
-                            label="Exclude NI"
-                            name="noNI"
-                            checked={inputs.noNI}
-                            onChange={handleInputChange}
-                        />
+                    <Form.Group className="mt-4 mb-4 d-flex">
```

Comment:
I kinda liked it without the column arrangement - one below the other. Having it with the rows and columns can cause the text to wrap, like this:
<img width="346" alt="Screenshot 2023-04-27 at 20 35 40" src="https://user-images.githubusercontent.com/29314003/234972573-6d0fceab-f620-4f3b-a8ee-75058a8666d0.png">
Do you mind reverting this part?

Reply:
Might be also worth adding some spacing on top of the Tax breakdown table (in Income Analysis) to keep it consistent.

Reply:
Yes, I'm happy to revert that... but I've also been working on some other layout tweaks to try and get everything working nicely on small screens.  I think these might avoid the problem altogether.  

Be warned that I did make some fairly bold decisions, like moving labels above form fields.  I personally think this works better and makes it easier to get the layout working well.  But I appreciate it might not be to your taste, and of course it's your project so I won't be at all offended if you reject them.